data/menu_data.rb

```diff
@@ -0,0 +1,9 @@
+MENU = {
```

Comment:
This is a good solution, having menu as a separate file. However does it have to be saved as a constant if it might change in the future?